package_builder/build_claw.sh

```diff
@@ -52,7 +52,7 @@ setupDefaults
 if [[ ${install_local} == "yes" ]]; then
     install_path_prefix_="${base_path}/claw/${resources_version}"
 else
-    install_path_prefix_="${install_dir}/claw//${resources_version}"
```

Comment:
I think a double slash `//` is the same as single slash. So I would expect this has no effect

Reply:
Indeed! I found the mistake however, the plan was still using the trick for tsa to use arolla and set COSMO_TESTENV=ON, and since then the buildenv was changed to fit the change that the slave arolla doesn't exist anymore. Do you want to test the script before merging?